### PR TITLE
update ci setup

### PR DESCRIPTION
**Pull request by tomerd**
motivation: 5.2 adoption, prepare for 5.3

changes:
* add 5.2 docker-compose setup
* add 5.3 docker-compose setup (placeholder)

### CODE REVIEW

File: .swiftformat [hunk 1]
```diff
@@ -1,5 +1,6 @@
 # file options
 
+--swiftversion 5.0
 --exclude .build
 
 # format options
```
Comment (yim-lee) [inline .swiftformat:6]: Can remove `--comments ignore`. It's deprecated.

File: docker/Dockerfile [hunk 2]
```diff
@@ -29,7 +30,7 @@ RUN chmod 755 $HOME/.tools/symbolicate-linux-fatal
 
 # swiftformat (until part of the toolchain)
 
-ARG swiftformat_version=0.40.12
+ARG swiftformat_version=0.44.0
```
Comment (yim-lee) [inline docker/Dockerfile:33]: `0.44.6`

Comment (ktoso) [reply docker/Dockerfile:33]: +1 yes please

Comment (ktoso) [inline docker/Dockerfile:33]:
```suggestion
ARG swiftformat_version=0.44.6
```

**Pull request merged**